.github/workflows/helm_charts_scalar.yml

```diff
@@ -91,20 +91,21 @@ jobs:
     strategy:
       matrix:
         k8s:
-          - v1.21.14
-          - v1.22.15
-          - v1.23.13
-          - v1.24.7
-          - v1.25.3
+          - v1.22.17
+          - v1.23.17
+          - v1.24.13
+          - v1.25.9
+          - v1.26.4
+          - v1.27.2
```

Comment:
I decided on these versions based on the supported k8s version in EKS and AKS.

* EKS
  https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
* AKS
  https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar